### DISCUSSION TITLE
CSV import with columns automatically created [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2187] [Discord Threads]
> 🙋‍♂️ **grndstt**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### this is doable from Quick Import which will be available in the next release  
#  
> 🙋‍♂️ **grndstt**  
#### 👍  is this a matter of days/weeks/months ?  
#  
> 👨‍🚀 **wingkwong**  
#### in a few days - we're wrapping up now  
#  
> 🙋‍♂️ **grndstt**  
#### amazing  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976753421841481748)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
